dashboard/s3.py

```diff
@@ -36,7 +36,7 @@ def __get_images(self):
         )
 
         for image in images['Contents']:
-            print image['Key']
+            print(image['Key'])
```

Comment:
This could become standard python logger content instead of a print statement.  To answer the question though.  Yes, please keep logging this.